<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_op1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>true</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>false</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_operacii">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Дизьюнкция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Коньюкция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эквиваленция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исключающее ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Импликация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Инверсия</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_op2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>true</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>false</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
